cleanlab/object_detection/filter.py

```diff
@@ -81,7 +82,7 @@ def find_label_issues(
         This is a list of ``N`` ``np.ndarray`` such that ``predictions[i]`` corresponds to the model prediction for the `i`-th image.
         For each possible class ``k`` in 0, 1, ..., K-1: ``predictions[i][k]`` is a ``np.ndarray`` of shape ``(M,5)``,
         where ``M`` is the number of predicted bounding boxes for class ``k``. Here the five columns correspond to ``[x1,y1,x2,y2,pred_prob]``,
-        where ``[x1,y1,x2,y2]`` are coordinates of the bounding box predicted by the model
+        where ``[x1,y1,x2,y2]`` are coordinates of the bounding box predicted by the model with respect to the y-coordinate axis
         and ``pred_prob`` is the model's confidence in the predicted class label for this bounding box.
 
         Note: Here, ``[x1,y1]`` corresponds to the coordinates of the bottom-left corner of the bounding box, while ``[x2,y2]`` corresponds to the coordinates of the top-right corner of the bounding box. The last column, pred_prob, represents the predicted probability that the bounding box contains an object of the class k.
```

Comment:
Note: In this context, the coordinate system follows the y-axis convention, where [x1, y1] denotes the coordinates of the bottom-left corner of the bounding box, and [x2, y2] represents the coordinates of the top-right corner of the bounding box. The last column, pred_prob, represents the predicted probability that the bounding box contains an object of the class k.

Reply:
I feel it's good to mention that we're iterating that we follow the y-coordinate axis in note as well